parser_test.go

```diff
@@ -501,6 +501,15 @@ func (s *testParserSuite) TestDMLStmt(c *C) {
 			SELECT * from tmp;
 		ROLLBACK;`, true, "START TRANSACTION; INSERT INTO `tmp` SELECT * FROM `bar`; SELECT * FROM `tmp`; ROLLBACK"},
 
+		// table statement
+		{"TABLE t", true, "TABLE `t`"},
+		{"TABLE t ORDER BY b", true, "TABLE `t` ORDER BY `b`"},
+		{"TABLE t LIMIT 3", true, "TABLE `t` LIMIT 3"},
+		{"TABLE t ORDER BY b LIMIT 3", true, "TABLE `t` ORDER BY `b` LIMIT 3"},
+		{"TABLE t ORDER BY b LIMIT 3 OFFSET 2", true, "TABLE `t` ORDER BY `b` LIMIT 3 OFFSET 2"},
+		{"TABLE t1 UNION TABLE t2", true, "TABLE `t1` UNION TABLE `t2`"},
+		{"TABLE t1 UNION SELECT * FROM t2", true, "TABLE `t1` UNION SELECT * FROM `t2`"},
```

Comment:
please also include these example test cases:

```sql
SELECT * FROM t1 WHERE a IN (TABLE t2);
TABLE t1 INTERSECT TABLE t2;
TABLE t1 EXCEPT TABLE t2;
```

and optionally:

```sql
INSERT INTO `test`.`b` TABLE `test`.`a`;
REPLACE INTO `test`.`b` TABLE `test`.`a`;
CREATE TABLE `test`.`b` TABLE `test`.`a`;
CREATE VIEW `test`.`b` AS TABLE `test`.`a`;
TABLE t1 INTO OUTFILE 'a.txt';
```

Reply:
Get~